src/components/album/Album.js

```diff
@@ -30,6 +31,9 @@ const AlbumTitle = styled('h3')({
 
 function Album({ album, wishListId }) {
   const [openDialog, setOpenDialog] = useState(false);
+  const reviewsRef = React.useRef(); // Created a ref to hold the function
```

Comment:
I'm not understanding the use-case for this line?

Reply:
just forgot to delete it, was trying to do smth